README.md

```diff
@@ -1,6 +1,8 @@
 # {{ name }}
 
-> a GitHub App built with [probot](https://github.com/probot/probot) that {{ description }}
+[![Build Status](https://travis-ci.org/probot/{{ name }}.svg?branch=master)](https://travis-ci.org/probot/{{ name }})
```

Comment:
I'm a little conflicted about having the travis badge right here, since I'm not sure we wanna "endorse" any specific CI service, even though we do use Travis. Do you think that's okay still?

Reply:
I think it's OK. I'll add a comment saying add your own CI service.